package.json

```diff
@@ -12,6 +12,8 @@
     "clean": "rimraf coverage dist tmp",
     "lint": "eslint . --ext .ts",
     "prebuild": "npm run lint",
+    "style": "prettier -w --config .prettierrc lib",
+    "style:check": "prettier -c --config .prettierrc lib",
```

Comment:
I'm not very convinced by the `style` word here. "Style" is too vague and can express anything in my point of view. Maybe just use the word `prettier`?

Reply:
Yeah, we can change it to `prettier`, as we use to have in between versions `v1.3.1-alpha.6` and `v2.0.0-alpha.0`.